src/ngx_http_lua_balancer.c

```diff
@@ -16,46 +16,103 @@
 #include "ngx_http_lua_directive.h"
 
 
+typedef struct {
+    lua_State                               *lua_vm;
+
+    uint32_t                                 crc32;
```

Comment:
This creates a 4-byte whole in the struct layout. Please check other similar places.